fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1940,6 +1940,12 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static boolean enable_shuffle_load = true;
 
+    /**
+     * Eliminate shuffle load by replicated storage
+     */
+    @ConfField(mutable = true)
```

Comment:
Why to need the configuration?

Reply:
replicated_storage is forced to open here, if there is a bug, we have to find a way to turn it off.

Reply:
If you are concerned about the `replicated_storage`, why not turn the `replicated_storage` off when there is a bug?